test/unit/Exception/OpenApiErrorTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Test\Unit\Exception;
+
+use OnMoon\OpenApiServerBundle\Exception\OpenApiError;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \OnMoon\OpenApiServerBundle\Exception\OpenApiError
+ */
+final class OpenApiErrorTest extends TestCase
```

Comment:
This test can be skipped at all, the subject class adds no new behavior to the base exception class